docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.navigation.scss

```diff
@@ -0,0 +1,101 @@
+// ICON CONTAINERS
+// ===============
+@mixin hb-menu-small-icon {
+  height: hb-calculate-rems(20px);
+  width: hb-calculate-rems(15px);
+
+  overflow: hidden;
+  text-indent: -9999px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+@mixin hb-menu-large-icon {
```

Comment:
same kind of rename here please :)

Reply:
orrrr you could make a generic mixin icon-wrapper that consolidates the logic between these two actually. That might be best!